src/components/NewMessageButtonHeader.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div class="header">
 		<ButtonVue :aria-label="t('mail', 'New message')"
-			type="primary"
+			type="secondary"
 			class="new-message-button"
```

Comment:
if the class is now unused please remove it. it's an old style class without BEM